vmm/src/lib.rs

```diff
@@ -2080,6 +2089,14 @@ impl Vmm {
             UpdateNetworkInterface(netif_update, sender) => {
                 Vmm::send_response(self.update_net_device(netif_update), sender);
             }
+            ConfigureVm(vmm_config, sender) => match self.configure(*vmm_config) {
+                Err(e) => {
+                    Vmm::send_response(Err(e), sender);
```

Comment:
If we're sending an Error response then closing the now _dirty_ process, we need to send a very explicit error that clearly notifies the user that the process needs to end.

Please create a custom `VmmActionError::SingleApiConfig(Box<VmmActionError>)` for this that holds the inner one. This way `Display` for it can specify that the single api config failed and Firecracker will exit, and also provide the original error as to why it failed.

Also add an `error!` log-entry (this is something requested by some customers for all API calls).

Reply:
Done.